lib/ex_bench/capturer.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ExBench.Capturer do
+  def capture(file, args \\ [trace_pattern: {:io, :format, 2}, count: 1]) do
+    {:ok, fh} = :file.open(file, [:append])
+
+    appender = fn {:trace, _pid, :call, {_m, _f, a}} ->
+      :file.write(fh, :io_lib.format("~p.~n", [a]))
+    end
+
+    :recon_trace.calls(args[:trace_pattern], args[:count], formatter: appender)
+
+    ### uh - how do I know when to close the file handle ? counting calls?
```

Comment:
I know recon sends some message about it because it responds in the shell. I agree, probably not a concern til ferd tells us :)

Reply:
if I give you a `count` arg I'll accidentally overwrite the default trace pattern arg. Better to just take options and replace with default if missing in the body I think.

Reply:
“reviewed and approved. I don’t know how to know when it’s done tracing but it tells you in the shell when it’s captured the requested number of traces” - good question ! You don’t - but when you close the shell - all those file handles you’ve allocated will be freed - also the file is opened in append mode, and there’s no file locking, so it’s not an issue if left open  and appended to by the next capture run